src/targeting.js

```diff
@@ -149,27 +167,46 @@ export function newTargeting(auctionManager) {
   }
 
   /**
-   * Get custom targeting keys for bids that have `alwaysUseBid=true`.
+   * Get custom targeting keys for bids`.
    */
-  function getAlwaysUseBidTargeting(adUnitCodes) {
+  function getCustomBidTargeting(adUnitCodes) {
     let standardKeys = getStandardKeys();
 
     return auctionManager.getBidsReceived()
-      .filter(adUnitsFilter.bind(this, adUnitCodes))
+      .filter(bid => adUnitCodes.includes(bid.adUnitCode))
```

Comment:
This entire code block is really unmanageable. Part of the reason for re-writing this module was to get better readability. I can't really tell what's going on here, and if there is a bug I'd never catch it from reading the code. It's very difficult to debug as well at runtime.

My suggestion would be to break each piece into separate functions that do one thing and give them meaningful names so that when they are composed the code is easier to understand.